### ISSUE TITLE
ENet-SAD TuSimple

### ISSUE THREAD

**Issue by rodrigoberriel** [cardwing/Codes-for-Lane-Detection#251]
First, thanks for providing the source-codes. Nice job!

I am interested in analyzing the predictions of your model in the TuSimple dataset (i.e., computing some stats and other metrics). In the [README](https://github.com/cardwing/Codes-for-Lane-Detection/blob/master/ENet-TuSimple-Torch/README.md) of the `ENet-TuSimple-Torch` directory, you say:

> Note that if you use ENet-Label-Torch (i.e., ENet-SAD) as the backbone, you can get around **96.64%** accuracy in TuSimple testing set.

However, it is not clear to me how to do that. In the `ENet-Label-Torch`, the pretrained file that was provided seems to be related to the CULane dataset. How could I get the predictions of ENet-SAD in the TuSimple that achieved 96.64%? Do you mind sharing the model trained on TuSimple with ENet-SAD?

**Comment by cardwing** [comment]
You can add the SAD module to ENet on TuSimple and then train the model. Eventually, you will obtain a model achieving 96.64% accuracy. As to the trained model, we only provide the trained ENet model. But you can train a model achieving 96.64% with the provided code.

**Comment by rodrigoberriel** [comment]
Would you happen to still have the trained model to share with us? Unfortunately, I don't have a server with many GPUs available right now. Anyway, if you don't have the trained model anymore, I'll give it a try and see how long it'll take to train on a single GPU.

In case you don't have it, feel free to close this issue. Thanks!

**Comment by MaddyThakker** [comment]
@rodrigoberriel were you able to achieve ~96% accuracy on the TuSimple testing set?

**Comment by rodrigoberriel** [comment]
@MaddyThakker no, I did not. I faced [other issues](https://github.com/cardwing/Codes-for-Lane-Detection/issues/208#issuecomment-592137693) while trying to do so. There are many missing pieces in the code that is available :(
